stories/info-label/info-label.stories.js

```diff
@@ -44,7 +52,14 @@ export const colors = () => `
     <span class="fd-info-label fd-info-label--accent-color-10">Info Label</span>
 </div>
 `;
-colors.storyName = 'Color Flavors';
+colors.storyName = 'Colors';
+colors.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: `Info label can be displayed in various colors, by adding the <code>fd-info-label--accent-color</code> modifier class to the element.
+`
```

Comment:
I think we need to have this as `<code>fd-info-label--accent-color-*</code>` and specify the options are numbers from 1 to 10. 
Someone can add just the modifier class <code>fd-info-label--accent-color</code> and it won't apply the color since you need to specify the number of the color. The `-*` part usually means a placeholder for a number or letter. @mikerodonnell89 what do you think about it? Does it make sense to add the `-*` part

Reply:
Maybe use the same naming convention as for {icon-name} ?

Reply:
Ah, you're right! I actually meant to put -* there!